languages/en-US/music/play.json

```
@@ -1,15 +1,15 @@
 {
     "DESCRIPTION": "Plays music for you!",
     "USAGE": "{{prefix}}play [song]",
-    "EXAMPLES": "{{prefix}}play Despacito",
-    "NO_VOICE_CHANNEL": "You must be connected in a voice channel!",
-    "VOICE_CHANNEL_CONNECT": "I can't connect into your voice channel!",
-    "MISSING_SONG_NAME": "Please enter a song name to search for (or a link)!",
-    "RESULTS_FOOTER": "Please enter a value to select one of the search results from 1 to 10.",
-    "NO_RESULT": "No result on Youtube!",
+    "EXAMPLES": "{{prefix}}play bad guy",
+    "NO_VOICE_CHANNEL": "You must be connected to a voice channel!",
+    "VOICE_CHANNEL_CONNECT": "I can't connect to your voice channel!",
+    "MISSING_SONG_NAME": "Please attach a song name or a link!",
+    "RESULTS_FOOTER": "Please select a song by entering a number between 1 and 10.",
```

Peut-être que cette formulation est plus précise ?
```suggestion
    "RESULTS_FOOTER": "Please select a song by sending a number between 1 and 10.",
```